src/features/views/components/ViewDataTable/columnTypes/LocalBoolColumnType.tsx

```diff
@@ -18,9 +18,15 @@ import useViewGrid, {
   UseViewGridReturn,
 } from 'features/views/hooks/useViewGrid';
 
+import messageIds from 'features/views/l10n/messageIds';
+
 export default class LocalBoolColumnType implements IColumnType {
-  cellToString(cell: boolean | null): string {
-    return String(!!cell);
+  cellToString(cell: boolean | null): string {    
+    if (cell == true) {
+      return messageIds.shareDialog.download.local_bool._defaultMessage;
```

Comment:
This will only ever use English. Using localization here will require the use of `getServerMessages` (from `src/core/i18n/server.ts`) and I'm not so sure how well that will work in this context since this code also runs on the frontend.

Reply:
Could a solution be to use the browser-language 🤔?

Reply:
> Could a solution be to use the browser-language 🤔?

Yes! The browser is one potential source. There's code in `scaffold()` (`src/utils/next.ts`) to figure out the language. The problem is not so much where to get it from (browser, `user.lang` or elsewhere) but how to get from there to here. I reckon we probably need to add a second parameter to `cellToString()` for the language, or an object including language (to make it more future proof) or something like that.